tests/Workflow/ValidateDocumentElementTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace tests\Libero\ContentApiBundle\Workflow;
+
+use FluentDOM;
+use Libero\ContentApiBundle\Exception\InvalidDocumentElement;
+use Libero\ContentApiBundle\Model\ItemId;
+use Libero\ContentApiBundle\Model\ItemVersionNumber;
+use Libero\ContentApiBundle\Model\PutTask;
+use Libero\ContentApiBundle\Workflow\ValidateDocumentElement;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Workflow\Event\Event;
+use Symfony\Component\Workflow\Marking;
+use Symfony\Component\Workflow\Transition;
+
+final class ValidateDocumentElementTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function it_passes_on_the_correct_document_element() : void
+    {
+        $document = FluentDOM::load('<item xmlns="http://libero.pub"/>');
+
+        $marking = new Marking();
+        $task = new PutTask('service', ItemId::fromString('id'), ItemVersionNumber::fromInt(1), $document);
+        $transition = new Transition('transition', 'place1', 'place2');
+
+        $validator = new ValidateDocumentElement();
+
+        $this->expectNotToPerformAssertions();
+
+        $validator->onValidate(new Event($task, $marking, $transition));
+    }
+
+    /**
+     * @test
+     */
+    public function it_fails_on_the_wrong_document_element() : void
+    {
+        $document = FluentDOM::load('<item xmlns="http://not.libero.pub"/>');
+
+        $marking = new Marking();
```

Comment:
can extract the next 4 lines to reduce duplication and make immediately clear the difference between the positive and negative cases (`$document`)